### PR TITLE
feat: implementation

### PR DESCRIPTION
**Pull request by madlabman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+### `oz-merkle-tree`
+
+**A Python library to generate Merkle trees and Merkle proofs.**
+
+This is project is compatible with the [OpenZeppelin StandardMerkleTree](https://github.com/OpenZeppelin/merkle-tree).
+
+#### Usage
```
Comment (vgorkavenko) [inline README.md]: would be great to have `Installation` routine section here

File: tests/test_tree.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+from typing import TypeAlias
```
Comment (vgorkavenko) [inline tests/test_tree.py:1]: just a thought for the future: would be great to see simple unit tests as doctests for functions

**Review by vgorkavenko — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+### `oz-merkle-tree`
+
+**A Python library to generate Merkle trees and Merkle proofs.**
+
+This is project is compatible with the [OpenZeppelin StandardMerkleTree](https://github.com/OpenZeppelin/merkle-tree).
```
Comment (skhomuti) [inline README.md]:
```suggestion
This project is compatible with the [OpenZeppelin StandardMerkleTree](https://github.com/OpenZeppelin/merkle-tree).
```

**Review by skhomuti — APPROVED** [review]

File: tests/test_tree.py (hunk 2 above)
Comment (madlabman) [reply tests/test_tree.py:1]: Maybe if we found well isolated functions for that.

File: README.md (hunk 1 above)
Comment (madlabman) [reply README.md]: Added

File: README.md (hunk 3 above)
Comment (madlabman) [reply README.md]: Done.

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**